Justfile

```diff
@@ -609,3 +609,6 @@ clean-pr-to-testrepo new_repo_name="test-actions-repo":
 
 # Alias for dev (full developer cycle: format → lint → test → build)
 alias cycle := dev
+ #Check license
+check-license-header:
+    pre-commit run addlicense --all-files
```

Comment:
@Tatianah3 Design question: why not have it run directly. Why have it run through pre-commit?